packages/react-core/src/components/Wizard/examples/WizardWithinModal.tsx

```diff
@@ -1,49 +1,48 @@
 import React from 'react';
-import { Button, Wizard, WizardStep, WizardHeader } from '@patternfly/react-core';
-import { Modal as ModalDeprecated, ModalVariant as ModalVariantDeprecated } from '@patternfly/react-core/deprecated';
+import { Button, Modal, ModalBody, ModalVariant, Wizard, WizardHeader, WizardStep } from '@patternfly/react-core';
 
-export const WizardWithinModal = () => {
+export const WizardWithinModal: React.FunctionComponent = () => {
   const [isModelOpen, setIsModalOpen] = React.useState(false);
 
   return (
     <>
       <Button onClick={() => setIsModalOpen(true)}>Show Modal</Button>
 
-      <ModalDeprecated
+      <Modal
         isOpen={isModelOpen}
-        showClose={false}
         aria-label="Wizard modal"
-        hasNoBodyWrapper
         onEscapePress={() => setIsModalOpen(false)}
-        variant={ModalVariantDeprecated.medium}
+        variant={ModalVariant.medium}
       >
-        <Wizard
-          height={400}
-          onClose={() => setIsModalOpen(false)}
-          title="In modal wizard"
-          header={
-            <WizardHeader
-              onClose={() => setIsModalOpen(false)}
-              title="Wizard in modal"
-              description="Simple wizard description"
-            />
-          }
-        >
-          <WizardStep name="Step 1" id="in-modal-step-1">
-            Step 1 content
-          </WizardStep>
-          <WizardStep name="Step 2" id="in-modal-step-2">
-            Step 2 content
-          </WizardStep>
-          <WizardStep
-            name="Review"
-            id="in-modal-review-step"
-            footer={{ nextButtonText: 'Finish', onNext: () => setIsModalOpen(false) }}
+        <ModalBody>
```

Comment:
Since `hasNoBodyWrapper` was previously being passed, we shouldn't need to use ModalBody in this example.